src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -295,13 +277,13 @@ export class BrowsingContextProcessor {
     params: Script.AddPreloadScriptParameters
   ): Promise<Script.AddPreloadScriptResult> {
     if (params.arguments !== undefined && params.arguments.length > 0) {
-      throw new Message.UnsupportedOperationException(
-        'add preload script arguments are not supported'
-      );
+      // TODO: Handle arguments.
+      throw new Error('add preload script arguments are not supported');
     }
 
+    // XXX: Stop using Set.
     const cdpTargets = new Set<CdpTarget>(
-      // TODO: flatten children and deduplicate.
+      // TODO: Flatten children.
       params.context === undefined || params.context === null
         ? this.#browsingContextStorage
             .getTopLevelContexts()
```

Comment:
Again, the unique target can be in non-top-level browsing context. What we really need here is all the targets, so we can walk through all the contexts and collect their targets into the set.

Reply:
Is it OK to do it in a follow up?

I am aware something needs to change here, hence the TODO.

I'll incorporate your comment in the TODO.

Reply:
Updated. Please let me know if you're OK with doing this in a follow-up PR.